### DISCUSSION TITLE
some weired process names started to appear

### DISCUSSION THREAD

**Discussion by molitona** [evilsocket/opensnitch#735] [Q&A]
Hi, lastly i started to see some weird names like the one in the picture below.
Initially, i ignored that and now i begin to be suspicious 

![ll](https://user-images.githubusercontent.com/35532337/188487284-6b998895-7f5c-49b3-b889-d82508d26005.png)

when i hover the cursor on the name it shows /bin/bash

i searched with locate command on my computer and no file have that name.

I'll update this post with other pictures in the future when i receive something like that.

what do you think ?

**Comment by NRGLine4Sec** [comment]
I have the same issue with some connections that seemed to come from grep, awk or https (for the apt binnary /usr/lib/apt/methods/https).

**Comment by gustavo-iniguez-goya** [comment]
version 1.6.0? If yes, I've also seen some oddities, like popularity-contest being called with setsid:

`/usr/bin/perl -w /usr/share/popularity-contest/...` vs `setsid /usr/bin/torify /usr/share/popularity-contest/...`

In order to debug these cases, set log level to DEBUG, and when a new pop-up appear copy the log file `/var/log/opensnitchd.log` and write down the process name + path to the binary + executed command.

>  I have the same issue with some connections that seemed to come from grep, awk or https (for the apt binnary /usr/lib/apt/methods/https).

This might be a different problem. If you have the oportunity follow the above steps, to analyze what's going on.

**Reply by NRGLine4Sec** [reply]
Hi @gustavo-iniguez-goya 
Yes, this is the version 1.6.0rc2.
I will create an appropriate issue with my findings by following the steps you mention if I encounter some new anomaly.

**Reply by molitona** [reply]
> version 1.6.0? If yes, I've also seen some oddities, like popularity-contest being called with setsid:
> 
> `/usr/bin/perl -w /usr/share/popularity-contest/...` vs `setsid /usr/bin/torify /usr/share/popularity-contest/...`
> 
> In order to debug these cases, set log level to DEBUG, and when a new pop-up appear copy the log file `/var/log/opensnitchd.log` and write down the process name + path to the binary + executed command.
> 
> > I have the same issue with some connections that seemed to come from grep, awk or https (for the apt binnary /usr/lib/apt/methods/https).
> 
> This might be a different problem. If you have the oportunity follow the above steps, to analyze what's going on.

Done. I'll post picture & log when it happens again.
